.github/workflows/CI_pull_request.yml

```diff
@@ -83,3 +83,51 @@ jobs:
           path: |
             pytest_junit_out.xml
             htmlcov/*
+  run-full-test-suite-ARM:
+    name: Run full test suite ARM64
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ["ubuntu-22.04-arm"]
+        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12"]
+        singularity: [
+          {provider: "apptainer", version: "latest"}
+          ]
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+      - name: Attempt to restore Singularity/Apptainer installer from cache
+        uses: actions/cache@v4
+        id: cache-singularity-installer
+        with:
+          path: ./singularity_installer.deb
+          key: ${{ matrix.os }}-${{ matrix.singularity.provider }}_installer-${{ matrix.singularity.version }}
```

Comment:
```suggestion
```
It seems that this caching of the singularity/apptainer installer is no longer needed/used since we just install the latest version from the ppa.